components/Navbar.js

```diff
@@ -399,6 +342,13 @@ const Navbar = (props) => {
     setSideMenu(state.isOpen);
   }
 
+  const pathname = router?.pathname ?? "";
+  const headerLabel = pathname.includes("notebook")
+    ? "Lab Notebook"
+    : pathname.includes("leaderboard")
+    ? "Leaderboard"
+    : deSlug(router?.query?.slug ?? "");
```

Comment:
We shouldn't need this. We only need to show title for the things on the left sidebar.
I like the approach though